includes/js/tec_dm_init.js

```diff
@@ -27,7 +27,7 @@ function tec_darkMode() {
     if(tec_darkMode_state == 'false') {
         var style = document.createElement("link");
         style.rel = "stylesheet";
-        style.href = "/wp-content/plugins/te-custom-mods/includes/css/tec_darkMode.css?v6";
+        style.href = "/wp-content/plugins/te-custom-mods/includes/css/tec_darkMode.css?v6.0";
```

Comment:
The build script requires the explicit `.0` on versions, which I happen to want to enforce anyways, so I manually added it here.